hugo/layouts/index.html

```diff
@@ -36,7 +36,9 @@ <h2 class="pl-2 pb-2">Welcome to the ACL Anthology!</h2>
 
     <aside class="alert alert-info acl-news mb-3" role="alert">
       <!-- The news message pulls from content/_index.md -->
+      {{ if (gt (time .PublishDate).Year 2018) }}
```

Comment:
Same here: why is the year hard-coded and what does it do? is it supposed to disable the news block if the news is disabled by setting an old date?  wouldn't it be better to check whether the news is empty?

Reply:
The date defaults to something very early, and only displays if its after 2018.

Reply:
The date is the one set in _index.md if I understand correctly.  I understood the code but disabling the MOTD by setting the date to something before 2019 seems non-obvious to me.  I am not opposed to merging it, it's just that maybe there exists an alternative solution which is more obvious on first sight (making long-time maintenance easier).

Reply:
The news isn't empty right now; only the date is. Removing the date line in `_index.md` just sets it to "January 0001". I'm happy to implement this more clearly but don't yet see how.

Reply:
Ah, now I understand what it is supposed to do.  In that case, I would just add a comment like `<!--to disable printing a date for the MOTD, set the date to < 2019-01-01 -->` in  `_index.md`.

Alternatively: get rid of the structure and write the date into the MOTD directly.